source/common/sequencer_impl.cc

```diff
@@ -144,7 +144,11 @@ void SequencerImpl::run(bool from_periodic_timer) {
       // TODO(oschaaf): Optionize performing this spin loop.
       platform_util_.yieldCurrentThread();
       spin_timer_->enableTimer(0ms);
-    }
+    } else if (idle_strategy_ == IdleStrategy::Sleep) {
+      // optionize sleep duration.
+      platform_util_.sleep(50us);
```

Comment:
I wonder how different this is to spinning when the sleep duration is this low. It's fine to merge, but worth thinking about.

Reply:
yes; I'm will write down some thoughts on this option (on why we have it, and when it makes sense to use it, and when one should not do so). And maybe the sleep time itself needs to be parameterized. But for, with the stock timing, it's an option to get us a higher precision compared to the 1 ms polling, while not maxing out cpu.